__tests__/app.test.js

```diff
@@ -163,4 +163,45 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/articles", () => {
+    test("status: 200 - responds with array of articles objects with author, title, article_id, topic, created_at, votes properties  ", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((res) => {
+          expect(res.body.articles).toHaveLength(12);
+          res.body.articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+              })
+            );
+          });
+        });
+    });
+    test("status: 200 - responds with array of articles sorted by date in descending order ", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((res) => {
+          expect(res.body.articles).toHaveLength(12);
+          expect(res.body.articles).toBeSortedBy("created_at", {
+            descending: true,
+          });
+        });
+    });
+    test(`status: 404 - responds with "path not found" msg for incorrect path`, () => {
```

Comment:
I'd say this test doesn't actually apply to this endpoint - this is your more general "PATH NOT FOUND" - the test itself looks great! But don't feel you need to implement for each endpoint individually